octopoes/bits/domain_owner_verification/domain_owner_verification.py

```diff
@@ -0,0 +1,32 @@
+from collections.abc import Iterator
+
+from octopoes.models import OOI
+from octopoes.models.ooi.dns.records import DNSNSRecord
+from octopoes.models.ooi.dns.zone import Hostname
+from octopoes.models.ooi.findings import Finding, KATFindingType
+
+INDICATORS = [
+    "DNS1.NAME-SERVICES.COM",
+    "DNS2.NAME-SERVICES.COM",
+    "DNS3.NAME-SERVICES.COM",
+    "DNS4.NAME-SERVICES.COM",
+    "DNS5.NAME-SERVICES.COM",
+    "NS1.REGISTRANT-VERIFICATION.ISPAPI.NET",
+    "NS2.REGISTRANT-VERIFICATION.ISPAPI.NET",
+    "NS3.REGISTRANT-VERIFICATION.ISPAPI.NET",
+]
+
+
+def run(nameserver: DNSNSRecord, additional_oois: list[Hostname], config: dict[str, str]) -> Iterator[OOI]:
+    """Checks to see if a domain has a specific set of dns servers which would indicate domain registrant verification.
+    https://support.dnsimple.com/articles/icann-domain-validation/
+    """
+    if DNSNSRecord.name_server_hostname in INDICATORS:
```

Comment:
Do we have to do some sanitization here (for casing in the text/ name) before searching, or can we assume the `name_server_hostname` is always in uppercase?

Reply:
we do now :)